### PR TITLE
solexaqa 3.1.7.1 (new formula)

### PR DESCRIPTION
**Pull request by lmrodriguezr**
- [x] Have you followed the [guidelines for contributing](https://github.com/brewsci/homebrew-bio/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/brewsci/homebrew-bio/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source FORMULA`, where `FORMULA` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict FORMULA` (after doing `brew install FORMULA`)?

-----

### CODE REVIEW

**Comment by sjackman** [comment]
Please rename this formula to `solexaqa` (remove `++`), since SolexaQA 2 is legacy and no longer supported. http://solexaqa.sourceforge.net

File: Formula/solexaqa++.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Solexaqaxx < Formula
+  desc "Quality statistics for second-generation sequencing data"
+  homepage "https://sourceforge.net/projects/solexaqa"
+  # doi "10.1186/1471-2105-11-485"
```
Comment (sjackman) [inline Formula/solexaqa++.rb]: Please change this line to…
```
# cite "http://doi.org/10.1186/1471-2105-11-485"
```
and move it after `sha256`.

File: Formula/solexaqa++.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+class Solexaqaxx < Formula
+  desc "Quality statistics for second-generation sequencing data"
+  homepage "https://sourceforge.net/projects/solexaqa"
```
Comment (sjackman) [inline Formula/solexaqa++.rb]: Please change this to http://solexaqa.sourceforge.net

**Comment by sjackman** [comment]
Thanks for the submission, Luis!

**Comment by sjackman** [comment]
Merged. Thanks for your contribution to Brewsci/bio, Luis! 🏅

**Comment by sjackman** [comment]
For future PRs, please squash the commits down to a single commit. You can use `git commit --amend` to update your most recent commit.

**Comment by lmrodriguezr** [comment]
Thank you for picking up Homebrew for bioinformatics!

**Pull request closed without merging**